testdata/dnn/onnx/generate_onnx_models.py

```diff
@@ -742,6 +743,7 @@ def forward(self, x):
 x = Variable(torch.randn(1, 2, 3, 4))
 save_data_and_model("reduceL2_subgraph", x, model)
 
+<<<<<<< HEAD
```

Comment:
Please resolve rebase artifacts

Reply:
BTW, consider enabling diff3 conflict style:
- https://stackoverflow.com/questions/27417656/should-diff3-be-default-conflictstyle-on-git
- `git config --global merge.conflictstyle diff3`

Reply:
/cc @ashishkrshrivastava

Reply:
Oh, I am sorry. I was working on another PR for last two days. So couldn't notice. 
I will correct above artifacts.

Reply:
Done !

Reply:
> BTW, consider enabling diff3 conflict style:
> 
> * https://stackoverflow.com/questions/27417656/should-diff3-be-default-conflictstyle-on-git
> * `git config --global merge.conflictstyle diff3`

@alalek Done! thanks